docs/user-guide.md

```diff
@@ -198,7 +200,7 @@ public interface List<E extends @Nullable Object> {...}
 
 If it were `interface List<E>` rather than `interface List<E extends @Nullable
 Object>` then `NumberList<E extends @Nullable Number> extends List<E>` would not be
-legal. That's because `List<E>` is short for `List<E extends Object>`. Inside
+legal. That's because `interface List<E>` is short for `interface List<E extends Object>`. Inside
```

Comment:
On first reading, it wasn't clear whether this refers to the `extends List<E>` part. Adding `interface` might be one option to fix this. Or instead of reusing `E` we could use different type variable names.